src/pages/en/reference/cli-reference.md

```diff
@@ -82,30 +147,26 @@ astro telemetry clear
 Make sure you add the `astro telemetry disable` command to your CI scripts.
 :::
 
-## Global Flags
+## Common flags
 
-### `--config path`
+### `--config <path>`
 
 Specifies the path to the config file. Defaults to `astro.config.mjs`. Use this if you use a different name for your configuration file or have your config file in another folder.
 
 ```shell
 astro --config config/astro.config.mjs dev
 ```
 
-### `--root path`
+### `--root <path>`
 
-Specifies the path to the project root. If not specified the current working directory is assumed to be the root.
+Specifies the path to the project root. If not specified, the current working directory is assumed to be the root.
 
 The root is used for finding the Astro configuration file.
 
 ```shell
 astro --root myRootFolder/myProjectFolder dev
 ```
 
-### `--reload`
-
-Clears the cache (dependencies are built within Astro apps).
-
 ### `--verbose`
 
 Enables verbose logging, which is helpful when debugging an issue.
```

Comment:
Could use some clarification on what verbose logging does

Reply:
I was trying to run `--verbose` myself and couldn't figure out the difference! An example or something to make that clear would be good, but this is definitely NWTWWHB!

Reply:
Seeing this too! Posted on Discord to gather feedback.